retrodep/gosource.go

```diff
@@ -533,19 +533,20 @@ func (src GoSource) RepoPathForImportPath(importPath string) (*RepoPath, error)
 
 	// No replacement found, use the import pth as-is
 	r, err := vcs.RepoRootForImportPath(importPath, false)
-	if err != nil && strings.ContainsRune(importPath, '_') {
-		// gopkg.in gives bad responses for paths like
-		// gopkg.in/foo/bar.v2/_examples/chat1
-		// because of the underscore. Remove it and try again.
+	if err != nil {
 		u := strings.Index(importPath, "_")
+		if u == -1 {
+			return nil, err
+		}
+		// go.pkg ... it and try again.
```

Comment:
Please use the full comment here. The `...` in my example was meant to represent all the text in between those words. :-)